spec/requests/distributions_requests_spec.rb

```diff
@@ -211,8 +211,9 @@
     describe "POST #create" do
       let!(:storage_location) { create(:storage_location, organization: organization) }
       let!(:partner) { create(:partner, organization: organization) }
+      let(:issued_at) { Time.current }
```

Comment:
Ditto here - we shouldn't need to specify this.

Reply:
Same as above. These are params for the request. `issued_at` now must be present. Unless we decide to coerce blank values into a default date/time. But if we do I think we need to communicate that to the user somehow.